src/photos.cs

```diff
@@ -1072,13 +1072,29 @@ interface PHPhotoLibrary {
 		[Export ("sharedPhotoLibrary")]
 		PHPhotoLibrary SharedPhotoLibrary { get; }
 
+		[Deprecated (PlatformName.iOS, 14, 0)]
+		[Deprecated (PlatformName.MacOSX, 10, 16)]
+		[Deprecated (PlatformName.TvOS, 14, 0)]
 		[Static, Export ("authorizationStatus")]
 		PHAuthorizationStatus AuthorizationStatus { get; }
 
+		[TV (14,0), Mac (10,16), iOS (14,0)]
+		[Static]
+		[Export ("authorizationStatusForAccessLevel:")]
+		PHAuthorizationStatus authorizationStatus (PHAccessLevel accessLevel);
```

Comment:
-> `GetAuthorizationStatus`
needs a verb/action (method) and can't start with a lower case character